modules/multi-arch-uninstalling-using-cli.adoc

```diff
@@ -21,13 +21,27 @@ You must delete the `ClusterPodPlacementConfig` object before uninstalling the M
 
 .Procedure
 
+. Get the `Subscription` object name for the Multiarch Tuning Operator by running the following command: 
++
+[source,terminal]
+----
+$ oc get subscription.operators.coreos.com
```

Comment:
```suggestion
$ oc get subscription.operators.coreos.com -n <namespace> <1>
---
<1> Replace `<namespace>` with the name of the namespace where you want to uninstall the Multiarch Tuning Operator.
```
@sr1kar99 this command line needs `-n <namespace>` too

Reply:
@lwan-wanglin 
Updated as per the suggestion.